app/models/solidus_avatax_certified/line.rb

```diff
@@ -95,7 +95,11 @@ def refund_lines
     end
 
     def refund_line
-      {
+      if @refund.refund_reason_id == Spree::RefundReason.find_by(name: 'Refund shipping fee')&.id
+        tax_code = Spree::TaxCategory.find_by(name: 'Shipping').&tax_code
```

Comment:
I don't like it. This is adding our app logic assumption in this code.. can we make it better?